### ISSUE TITLE
Stats for dropped requests

### ISSUE THREAD

**Issue by tescophil** [AdguardTeam/AdGuardHome#4367]
### Problem Description
This is not related to a problem

### Proposed Solution
I would like to see stats for dropped requests.

### Alternatives Considered
Nothing I could think of

### Additional Information
So, when I look at the stats to see how many DNS requests have been processed by my AdGuard Home instance I don't get a full picture. This is because no stats are displayed for 'dropped' requests. These requests could be:

1) IPv6 requests dropped because the 'Disable resolving of IPv6 addresses' option is set under the 'DNS Settings'.
2) Requests from unauthorised clients (either not in the Allowed Clients list, or in the disallowed clients list)
3) Requests for unauthorised domains

I would be useful to see the overall (if not detailed) stats for these requests which are still received and processed by AdGuard Home, but not logged (as far as I know).

**Comment by fernvenue** [comment]
Of course you can, just edit `./AdGuardHome.yaml` and make sure the option `verbose` is `true`, and set a path you want by `log_file`, you will get all the information, include dropped requests.

**Comment by tescophil** [comment]
Thanks for the reply. I'm assuming this just turns on verbose logging, as opposed to displaying stats about dropped requests in the UI ?  (which is what I would like to see...)

**Comment by fernvenue** [comment]
> Thanks for the reply. I'm assuming this just turns on verbose logging, as opposed to displaying stats about dropped requests in the UI ? (which is what I would like to see...)

Okay I see, but I still don’t think that’s a good idea. Drop means throw away and **do nothing** include logging. Most applications behave this way, even iptables. Another thing, we often use Disallowed domains to drop and ignore some requests than reject by blocking rule, just because we don't wanna see them in web UI. Since we can see detailed records by turning on the log when we need it, why change?

**Comment by tescophil** [comment]
It's not a big deal, but I just think the current stats are somewhat misleading. The total number of requests shown on the UI is not the total number of requests received by the system, its as simple as that. Why can't we have figures for received, blocked and dropped on the UI ? I'm interested to see for example, how many IPv6 requests there are coming from devices on my network, or if some 'bad actor' in my network starts spamming the server with bad requests that are dropped, I have no way of seeing this information on the UI.

It also makes it difficult for a 'non technical' user to diagnose problems. eg. If the encryption certificate expires, then any DoT or DoH requests sent to the server are dropped, and currently this information is not visible in the UI. You have to get to machine, stop the service, edit the config, restart, and then wade through the deluge of information that is subsequently produced.., not something your 'average' user is going to do.

If the data is already there, why not have an option to display it ?

**Comment by fernvenue** [comment]
> It's not a big deal, but I just think the current stats are somewhat misleading. The total number of requests shown on the UI is not the total number of requests received by the system, its as simple as that. Why can't we have figures for received, blocked and dropped on the UI ? I'm interested to see for example, how many IPv6 requests there are coming from devices on my network, or if some 'bad actor' in my network starts spamming the server with bad requests that are dropped, I have no way of seeing this information on the UI.
> 
> It also makes it difficult for a 'non technical' user to diagnose problems. eg. If the encryption certificate expires, then any DoT or DoH requests sent to the server are dropped, and currently this information is not visible in the UI. You have to get to machine, stop the service, edit the config, restart, and then wade through the deluge of information that is subsequently produced.., not something your 'average' user is going to do.
> 
> If the data is already there, why not have an option to display it ?

Yes. That might be good to be able to provide options to allow viewing more logs, and not changing the default behavior. Let's see what the dev will say.

**Comment by sannidhyaroy** [comment]
+1
This feature would be helpful!

**Comment by hexclann** [comment]
+1
